lib/cron-expression/CronExpression.php

```diff
@@ -94,7 +94,8 @@ public function __construct($expression, CronExpression_FieldFactory $fieldFacto
      */
     public function setExpression($value)
     {
-        $this->cronParts = preg_split('/\s/', $value, -1, PREG_SPLIT_NO_EMPTY);
+        $cronParts = preg_split('/\s/', $value, -1, PREG_SPLIT_NO_EMPTY);
+        $this->cronParts = is_array($cronParts) ? $cronParts : [];
```

Comment:
For consistency with WP coding standards:

```suggestion
        $this->cronParts = is_array( $cronParts ) ? $cronParts : array();
```